openpype/hosts/houdini/plugins/publish/extract_opengl.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+import pyblish.api
+import openpype.api
+from openpype.hosts.houdini.api.lib import render_rop
+
+import hou
+
+
+class ExtractOpenGL(openpype.api.Extractor):
+
+    order = pyblish.api.ExtractorOrder
+    label = "Extract OpenGL"
+    families = ["review"]
+    hosts = ["houdini"]
+    optional = True
+
+    def process(self, instance):
+
+        ropnode = hou.node(instance.data["instance_node"])
+
+        # Get the filename from the filename parameter
+        # `.evalParm(parameter)` will make sure all tokens are resolved
+        output = ropnode.evalParm("picture")
```

Comment:
I guess it's quite unlikely this could change to another rop but would it be better to use the util function?
```
output = get_output_parameter(ropnode).eval()`
```